.gitattributes

```
@@ -21,8 +21,9 @@
 *.exe binary
 
 # Remove files for archives generated using `git archive`
-codeception.yml export-ignore
+architecture-baseline.json export-ignore
 dependency.json export-ignore
```

```suggestion
dependency.json export-ignore
phpstan.json export-ignore
phpstan.neon export-ignore
```

Please add missing files related to phpstan